src/apis/image/queries.ts

```diff
@@ -0,0 +1,7 @@
+import { useMutation } from '@tanstack/react-query'
+import { postUploadImages } from './apis'
+
+export const useUploadImagesQuery = () =>
```

Comment:
```suggestion
export const useUploadImagesMutation = () =>
```
Mutation으로 되면 좋을 것 같습니다!

Reply:
[변경완료](https://github.com/price-offer/offer-fe/pull/179/commits/17abbec909f742c99514d4e2c6f59e8da13e77e8) !